packages/mask/src/components/shared/VerifyWallet/Steps.tsx

```diff
@@ -16,9 +16,17 @@ import classNames from 'classnames'
 import { useEffect } from 'react'
```

Comment:
I suggest replacing the component to be a dumb component. All event handle and data should be obtained from the outside.